Libraries/LibJS/Tests/run-tests

```diff
@@ -9,24 +9,22 @@ else
     export UBSAN_OPTIONS=print_stacktrace=1
 fi
 
-extra_args="$*"
-
-pass_count=0
-fail_count=0
-count=0
+(( pass_count = 0 ))
+(( fail_count = 0 ))
+(( count = 0 ))
```

Comment:
Is shellcheck actually complaining about this? It looks really weird

Reply:
Yes, it is.

Reply:
We could keep this..
```bash
pass_count=0
fail_count=0
count=0
```
..it's not complaining about that, declaring the variable between `((` and `))` declares the variable as a number instead of as a string though. The script would work either way.

Should I revert this change?

Reply:
But these are not `let`?

Reply:
Yes please revert it. I am already kinda skeptical of the value of `shellcheck` for us, and using it as a justification to make code less readable seems like a step in the wrong direction :)

Reply:
Ok, I've done it.